CONTRIBUTING.adoc

```diff
@@ -53,6 +56,12 @@ openssl pkcs8 -topk8 -inform PEM -outform PEM -in ORIGINAL_KEY.pem -out CONVERTE
 
 NOTE: You need to change `ORIGINAL_KEY` and `CONVERTED_KEY` with real values.
 
+This final file is the one to point to in `GITHUB_APP_PRIVATE_KEY_PATH` environment variable.
+
+Finally, you need to install the GitHub APP on your GitHub user.
+In the GitHub App configuration page, find the *Install App* like on the left side menu.
```

Comment:
Optional
```suggestion
In the GitHub App configuration page, find the *Install App* option on the left side menu.
```

Reply:
it was link, not like. thanks for finding that typo.